os/fs/tmpfs/Kconfig

```diff
@@ -0,0 +1,74 @@
+#
+# For a description of the syntax of this configuration file,
+# see kconfig-language at https://www.kernel.org/doc/Documentation/kbuild/kconfig-language.txt
+#
+
+config FS_TMPFS
+	bool "TMPFS file system"
+	default n
+	depends on !DISABLE_MOUNTPOINT
+	select FS_READABLE
+	select FS_WRITABLE
+	---help---
+		Enable TMPFS filesystem support
+
+config FS_AUTOMOUNT_TMPFS
```

Comment:
you can keep this Kconfig entry inside 
if FS_TMPFS

Reply:
okay